wolox-books/src/components/BooksList.vue

```diff
@@ -1,6 +1,10 @@
 <template lang='pug'>
 .books-container
-  router-link.books-list(v-for='book in allBooks' :key='book.id' to='/')
+  router-link.books-list(
+    v-for='book in allBooks'
+    :key='book.id'
+    :to="'/books/' + book.id"
```

Comment:
Try another way of setting the id, setting it with this type of string is not really clean